testing/src/test/java/io/stargate/it/cql/Lz4CompressionTest.java

```diff
@@ -0,0 +1,23 @@
+package io.stargate.it.cql;
+
+import com.datastax.oss.driver.api.core.config.OptionsMap;
+import com.datastax.oss.driver.api.core.config.TypedDriverOption;
+import io.stargate.it.storage.ClusterConnectionInfo;
+import org.junit.jupiter.api.Test;
+
+public class Lz4CompressionTest extends AbstractCompressionTest {
```

Comment:
Do we need `CQLTest.compressionTest` now?

Reply:
There is some overlap between `CQLTest` and these new tests, I should definitely do some cleanup.

Reply:
Looking at this PR, I think `CQLTest` can be deleted entirely.